test/e2e/storage/persistent_volumes-local.go

```diff
@@ -461,6 +508,18 @@ func createLocalPod(config *localTestConfig, volume *localTestVolume) (*v1.Pod,
 	return framework.CreatePod(config.client, config.ns, []*v1.PersistentVolumeClaim{volume.pvc}, false, "")
 }
 
+func createAndMountTmpfsMntPnt(config *localTestConfig, dir string) {
+	By(fmt.Sprintf("Creating tmpfs mount point on node %q at path %q", config.node0.Name, dir))
+	err := framework.IssueSSHCommand(fmt.Sprintf("mkdir -p %q && sudo mount -t tmpfs -o size=1m tmpfs %q", dir, dir), framework.TestContext.Provider, config.node0)
```

Comment:
`"-o size=1m tmpfs %q"` - the last `tmpfs` should be a random string here, otherwise it may clash with already existing tmpfs named `tmpfs`.